libpod/oci_conmon_exec_linux.go

```diff
@@ -387,7 +387,7 @@ func (r *ConmonOCIRuntime) startExec(c *Container, sessionID string, options *Ex
 		finalEnv = append(finalEnv, fmt.Sprintf("%s=%s", k, v))
 	}
 
-	processFile, err := prepareProcessExec(c, options.Cmd, finalEnv, options.Terminal, options.Cwd, options.User, sessionID)
+	processFile, err := prepareProcessExec(c, options.Cmd, options.CapAdd, finalEnv, options.Terminal, options.Cwd, options.User, sessionID)
```

Comment:
Can you change this interface to just pass in the options?

Reply:
done